docs/docs-beta/docs/guides/external-assets.md

```diff
@@ -1,16 +1,20 @@
 ---
 title: Representing external data sources with external assets
 sidebar_position: 80
-sidebar_label: 'External data assets'
+sidebar_label: 'External data sources'
 ---
 
-One of Dagster's goals is to present a single unified lineage of all of the data assets in an organization. This can include assets orchestrated by Dagster and assets orchestrated by other systems.
+One of Dagster's goals is to present a single unified lineage of all of the data assets in an organization, even if those assets are orchestrated by systems other than Dagster.
 
-**External assets** enable you to model assets orchestrated by other systems natively within Dagster's Asset catalog, and create new data assets downstream of these external assets.
+With **external assets**, you can model assets orchestrated by other systems natively within Dagster, ensuring you have a comprehensive catalog of your organization's data. You can also create new data assets downstream of these external assets.
 
-External assets differ from native Dagster assets in that Dagster can't materialize them directly or put them on a schedule. Instead, an external system must inform Dagster of when an external asset is updated.
+How are external assets different than native Dagster assets? The answer: Dagster can't materialize external assets directly or put them in a schedule. In these cases, an external system must inform Dagster when an external asset is updated.
 
-Examples of external assets could be files in a data lake that are populated by a bespoke internal tool, a CSV file delivered daily by SFTP from a partner, or a table in a data warehouse populated by another orchestrator.
+For example, external assets could be:
+
+- Files in a data lake that are populated by a bespoke internal tool
+- A CSV file delivered daily by SFTP from a partner
+- A table in a data warehouse populated by another orchestrator
 
 ## What you'll learn
```

Comment:
Missed this originally, @erinkcochran87 , but we will want to remove this section.